Resources/Textures/Objects/Fun/spraycans.rsi/meta.json

```diff
@@ -31,6 +31,15 @@
     {
       "name": "clown_cap"
     },
+    {
+      "name": "cap_clown"
+    },
+    {
+      "name": "cap_death"
+    },
+    {
+      "name": "cap_spray"
+    },
```

Comment:
Are all the sprites you've added from tgstation?

Reply:
they already existed in the files.